src/common/header/components/GitHubButton.tsx

```diff
@@ -0,0 +1,18 @@
+import { IconButton } from '@chakra-ui/button';
+import { AiFillGithub } from 'react-icons/ai';
+
+export function GitHubButton(): JSX.Element {
+  return (
+    <IconButton
```

Comment:
One thing that would be nice would be if it got a little darker when the user hovers over the button. You can see an example of how to do that with props [here](https://github.com/VikeLabs/courseup/blob/main/src/common/header/components/Banner.tsx#L15-L18)

Reply:
oh true, hadn't thought of that! I've [committed a change for that](https://github.com/VikeLabs/courseup/pull/257/commits/f80d9a0b25ee2e4741278f3082d08fd3f7b037ad), let me know what you think about how dark it's getting and lmk if you want me to modify it